digits/model/forms.py

```diff
@@ -300,10 +312,11 @@ def validate_lr_multistep_values(form, field):
     )
 
     def validate_custom_network_snapshot(form, field):
-        if form.method.data == 'custom':
-            for filename in field.data.strip().split(os.path.pathsep):
-                if filename and not os.path.exists(filename):
-                    raise validators.ValidationError('File "%s" does not exist' % filename)
+        pass
+#        if form.method.data == 'custom':
+#            for filename in field.data.strip().split(os.path.pathsep):
+#                if filename and not os.path.exists(filename):
+#                    raise validators.ValidationError('File "%s" does not exist' % filename)
```

Comment:
So the problem is with tensorflow files where it has the .data-00000-of-00001 problem